### PR TITLE
Edit 3D collision shapes

### PR DESCRIPTION
**Pull request by NathanLovato**
- Added a bullet list in the intro to summarize what's in the article
- Passive -> direct voice
- Remove parens in sentences
- Tweak some sentences' wording
- Fix typos
- Minor grammar fixes
- Add missing Oxford commas

### CODE REVIEW

**Comment by NathanLovato** [comment]
Let me know if you'd prefer me to put comments and suggestions, but after doing it for #4058, I think it's much more efficient if we work like that: one writing pass, one person editing. People are welcome to do that with my PRs too, merging and editing directly.

File: tutorials/physics/collision_shapes_3d.rst [hunk 1]
```diff
@@ -25,16 +30,16 @@ Godot provides the following primitive collision shape types:
 - :ref:`class_BoxShape`
 - :ref:`class_SphereShape`
 - :ref:`class_CapsuleShape`
-- :ref:`class_CylinderShape` (only when using the Bullet physics engine)
+- :ref:`class_CylinderShape`. It is only available when when using the Bullet
```
Comment (pycbouh) [inline tutorials/physics/collision_shapes_3d.rst:33]: A typo 🙂

Comment (NathanLovato) [reply tutorials/physics/collision_shapes_3d.rst:33]: Where is the typo?

Comment (NathanLovato) [reply tutorials/physics/collision_shapes_3d.rst]: Thx!

**Comment by Calinou** [comment]
> Let me know if you'd prefer me to put comments and suggestions, but after doing it for #4058, I think it's much more efficient if we work like that: one writing pass, one person editing. People are welcome to do that with my PRs too, editing directly.

It's fine to do it this way too :slightly_smiling_face:

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**